### ISSUE TITLE
手动修改 PNNX 模型脚本以进行动态形状推理相关问题

### ISSUE THREAD

**Issue by sumo-s** [nihui/ncnn-android-yolov8#15]
自动生成的yolov8n_pnnx.py中出现的foward函数中模型架构为v_164 = self.model_22_cv3_2_2(v_163)
        v_165 = torch.cat((v_159, v_164), dim=1)
        v_166 = [v_143.size(0), 144, -1]
        v_167 = v_143.view(*v_166)
        v_168 = v_154.view(*v_166)
        v_169 = v_165.view(*v_166)
        v_170 = torch.cat((v_167, v_168, v_169), dim=2)
        v_171, v_172 = torch.split(tensor=v_170, dim=1, split_size_or_sections=(64,80))
        v_173 = [v_171.size(0), 4, 16, v_171.size(2)]
        v_174 = v_171.view(*v_173)
        v_175 = torch.transpose(input=v_174, dim0=2, dim1=1)
        v_176 = F.softmax(input=v_175, dim=1)
        v_177 = self.model_22_dfl_conv(v_176)
        v_178 = [v_171.size(0), 4, v_171.size(2)]
        v_179 = v_177.view(*v_178)
        v_180 = torch.unsqueeze(input=v_132, dim=0)
        v_181, v_182 = torch.chunk(input=v_179, chunks=2, dim=1)
        v_183 = (v_180 - v_181)
        v_184 = (v_180 + v_182)
        v_185 = ((v_183 + v_184) / 2)
        v_186 = (v_184 - v_183)
        v_187 = torch.cat((v_185, v_186), dim=1)
        v_188 = (v_187 * v_131)
        v_189 = F.sigmoid(input=v_172)
        v_190 = torch.cat((v_188, v_189), dim=1)
        return v_190

与示例中的修改前架构有区别，老师了解是哪里出现的问题吗？应当如何修改？
